### ISSUE TITLE
项目业务专业性较强：请问大佬，该系统是否需要配合kettle8.3的客户端使用 ，不能独立使用，非常好的项目

### ISSUE THREAD

**Issue by arraycto** [Selegant/etl-admin#5]
项目业务专业性较强：请问大佬，该系统是否需要配合kettle8.3的客户端使用 ，不能独立使用？
1.没有找到新建执行任务的页面
2.请问下图的
![image](https://user-images.githubusercontent.com/37463247/119925892-e017d000-bfa8-11eb-80ff-bf0f69610a7d.png)
任务如何得来的，是通过kettle客户端同步到mysql中，然后咱们系统从mysql中查到的么，。

目前业务无法实现闭环。感谢大佬~

**Comment by Selegant** [comment]
> 项目业务专业性较强：请问大佬，该系统是否需要配合kettle8.3的客户端使用 ，不能独立使用？
> 1.没有找到新建执行任务的页面
> 2.请问下图的
> ![image](https://user-images.githubusercontent.com/37463247/119925892-e017d000-bfa8-11eb-80ff-bf0f69610a7d.png)
> 任务如何得来的，是通过kettle客户端同步到mysql中，然后咱们系统从mysql中查到的么，。
> 
> 目前业务无法实现闭环。感谢大佬~

是的 这个需要配合Kettle客户端使用 将Kettle的作业以及转换存储到资源库中 然后在系统里面配置kettle的资源库 连接成功后就可以同步Kettle客户端里面存储的任务和转换 这个项目只是更加方便的调用Kettle的任务和转换 对于Kettle的功能没有增强 还有就是Kettle的工作流页面暂时还没有办法直接集成到系统里面 所以只能通过原有的Kettle客户端进行任务编写

**Comment by arraycto** [comment]
有点没想通的是，.ktr的后缀程序，存储在哪里，数据库也没发现有相关的啊，求大佬指点一下业务逻辑，给一个helloword的例子

**Comment by arraycto** [comment]
如图，同步没有任务数据。




![image](https://user-images.githubusercontent.com/37463247/120295818-18e1dd00-c2fa-11eb-9766-640eddfd80b5.png)
